src/missing_prices.py

```diff
@@ -128,14 +135,14 @@ def run_missing_prices() -> None:
     print(f"Fetched {len(tokens)} traded tokens from Dune without prices")
     found, res = 0, []
     for token in tokens:
-        if token["address"].lower() in coins:
-            paprika_data = coins[token["address"].lower()]
+        if token["address"] in coins:
+            paprika_data = coins[token["address"]]
             dune_row = (
-                str(paprika_data["id"]),
+                paprika_data["id"],
                 "ethereum",
-                str(paprika_data["symbol"]),
-                str(paprika_data["address"].lower()),
-                int(token["decimals"]),
+                paprika_data["symbol"],
+                paprika_data["address"],
+                token["decimals"],
```

Comment:
same here, `token.decimals`?

Reply:
Refer to discussion [here](https://github.com/cowprotocol/data-misc/pull/21#discussion_r1005397164).